TrainingIOS/ViewController.swift

```diff
@@ -50,7 +50,49 @@ class ViewController: UIViewController, UITextFieldDelegate {
         guard let password = txtPassword.text, password.characters.count > 6 else{
             return showAlert(title: "Notification", message: "password too short. Please input password > 6 characters")
         }
-        present( UIStoryboard(name: "MainApp", bundle: nil).instantiateViewController(withIdentifier: "MainApp") as! UITabBarController, animated: true, completion: nil)
+        
+        let postString = ["username": email, "password": password]
+        var request = URLRequest(url:URL(string:"https://us-central1-travelworld-5d555.cloudfunctions.net/v1/user/login")!)
+        request.httpMethod = "POST"
+        
+        request.addValue("application/json", forHTTPHeaderField: "Content-Type")
+        
+        request.httpBody = try! JSONSerialization.data(withJSONObject: postString)
+        
+        let session = URLSession.shared
+        
+        
+        session.dataTask(with: request) { data, response, err in
+            guard(err == nil) else {
+                print("\(String(describing: err))")
+                return
+            }
+            guard data != nil else{
+                print("no data return")
+                return
+            }
+            
+            let parseResult: [String:AnyObject]!
```

Comment:
let parseResult: [String:AnyObject]?
vì nếu trong do bị fail thì ở catch chú chỉ đơn thuần return chứ k gán gì cho parseResult, nên để optional sẽ an toàn